src/backend/aspen/workflows/nextstrain_run/tests/test_export.py

```diff
@@ -126,6 +127,11 @@ def test_build_config(mocker, session, postgres_database):
         assert nextstrain_config["files"]["description"].endswith(".md")
         assert len(sequences.splitlines()) == 20  # 10 county samples @2 lines each
         assert len(metadata.splitlines()) == 11  # 10 samples + 1 header line
+        # Test that we are careful with printing data from our models
+        metadata_reader = csv.DictReader(StringIO(metadata), delimiter="\t")
+        for row in metadata_reader:
+            for value in row.values():
+                assert not value.startswith("<aspen.database.models")
```

Comment:
Can we also add a positive test so we can see that the ISL/Accessions *do* appear?